app/views/hyrax/homepage/_featured.html.erb

```diff
@@ -1,11 +1,9 @@
 <% presenter = featured.presenter %>
-<li class="featured-item" data-id="<%= presenter.id %>">
-  <div class="main row">
-    <div class="col-sm-3">
-      <%= render_thumbnail_tag presenter, { width: 90 } %>
-    </div>
-    <div class="col-sm-9">
-      <%= render 'hyrax/homepage/featured_fields', featured: presenter %>
-    </div>
-  </div>
-</li>
+<tr class="featured-item" data-id="<%= presenter.id %>">
```

Comment:
Why are you switching to a table layout? Semantically, this is a list of featured works, right?